libraries/math/src/approximations.rs

```diff
@@ -67,4 +92,19 @@ mod tests {
             check_square_root(a as u128);
         }
     }
+
+    fn check_normal_cdf_f32(argument: f32) {
+        let result = f32_normal_cdf(argument).unwrap();
+        let abs_difference: f32 = (result - 0.841_344_8).abs();
+        dbg!(result);
+        assert!(abs_difference <= 0.001);
+    }
+
+    #[test]
+    fn test_normal_cdf_f32() {
+        let test_arguments: [f32; 1] = [1.0];
```

Comment:
This is a great opportunity to use the excellent `proptest` crate! Since the paper claims a maximum difference of `.0001922`, you can compare the output against `.5 * libm::erff(-1 * x / f32::SQRT_2)`, which should be the closest number. Check out libm https://rust-random.github.io/rand/libm/index.html.  Otherwise, you can check out how proptest is used right above this.